### PR TITLE
update package.json

### PR DESCRIPTION
**Pull request by diasdavid**
Created this PR to make travis run again on clean master to check the errors that @nginnever mentioned

### CODE REVIEW

**Comment by diasdavid** [comment]
yeah, this is weird: https://travis-ci.org/ipfs/js-ipfs/jobs/128359184#L2133-L2136

**Comment by diasdavid** [comment]
@dignifiedquire you might want to take a look. This was introduced with the update on libp2p-swarm: https://github.com/diasdavid/js-libp2p-swarm/commit/85a064765a15fe3ab64f8b03fcb6de5fe72097b1

Probably the problem was always there, but now it is really easy to see

**Comment by diasdavid** [comment]
> 18:57 <voxelot> daviddias: everything before this commit is passing for me 
18:57 <voxelot> https://github.com/ipfs/js-ipfs/commit/1251dea923eca703d597ad060da58ca518c49898

**Comment by nginnever** [comment]
>18:57 daviddias: everything before this commit is passing for me 
18:57 1251dea

I take that back, I started hard reseting commits again one at a time and the commit before this one through the spdy error as well.

**Comment by diasdavid** [comment]
@nginnever wanna check with libp2p-swarm 0.12.5 ?

**Comment by nginnever** [comment]
Downloaded the archived libp2p-swarm v0.12.5 and npm linked it to the js-ipfs repos, seems to fix the issue.

**Comment by nginnever** [comment]
Changed libp2p-swarm to throw transport close errors
 ```
async.each(
  Object.keys(this.transports),
  (key, cb) => this.transports[key].close(cb),
  callback
)
```

and received 
```
Error: there are no listeners
```
In js-ipfs libp2p core swarm.close

File: src/core/index.js [hunk 1]
```diff
@@ -27,7 +26,8 @@ function IPFS (repoInstance) {
     throw new Error('Must be instantiated with new')
   }
 
-  if (!(repoInstance instanceof IPFSRepo)) {
+  if (typeof repoInstance === 'string' ||
+      repoInstance === undefined) {
```
Comment (diasdavid) [inline src/core/index.js]: I've noticed that on Travis, the "isDaemonOn" always yielded `true`, making the 'cli offline' tests work online. Checking if this was the issue. @nginnever @dignifiedquire

Comment (diasdavid) [inline src/core/index.js]: Yep, it fixed the 'weird fails' on the block tests : )

**Comment by diasdavid** [comment]
Hopefully, this will give us a better answer https://github.com/indutny/spdy-transport/issues/26 (probably the right answer is the one that is most simply, we need to check for `.on('error') in every muxer we create`

File: src/core/index.js (hunk 1 above)
Comment (nginnever) [inline src/core/index.js]: I put these changes into the files PR.

**Comment by diasdavid** [comment]
GREEN :D

**Pull request merged**